### PR TITLE
Tweak vk_validation_stats script

### PR DESCRIPTION
**Pull request by daveh-lunarg**
Improve the error text filtering to get rid of more &html;escape;&codes;.

Allow the vuid lookup to report implicit checks & line numbers.

### CODE REVIEW

File: scripts/vk_validation_stats.py [hunk 1]
```diff
@@ -653,13 +657,23 @@ def main(argv):
...
+                for line in line_list:
+                    print('    => %s' % line)
             else:
                 print('  Not implemented.')
             if get_vuid_status in val_tests.explicit_vuids:
                 print('  Has a test!')
                 test_list = val_tests.vuid_to_tests[get_vuid_status]
                 for test in test_list:
                     print('    => %s' % test)
+            elif get_vuid_status in val_tests.implicit_vuids:
```
Comment (tobine) [inline scripts/vk_validation_stats.py]: This looks to have exact same code as "if" case above. If that's true should just make it a compound "if" clause:

`if (get_vuid_status in val_tests.explicit_vuids) or (get_vuid_status in val_tests.implicit_vuids):`

Comment (jzulauf-lunarg) [reply scripts/vk_validation_stats.py]: :+1:

**Comment by daveh-lunarg** [comment]
Changes pushed

File: scripts/vk_validation_stats.py [hunk 2]
```diff
@@ -204,8 +204,12 @@ def read(self):
                         self.implicit_vuids.add(vuid_string)    # otherwise, implicit
                         vtype = 'implicit'
                     vuid_text = ventry['text']
-                    stripped = re.sub(self.re_striptags, '', vuid_text) # strip tags
-                    stripped = html.unescape(stripped) # unescape html literals (only partially works - because asciiDoctor?)
+                    #if 'amp;' in vuid_text:
```
Comment (mark-lunarg) [inline scripts/vk_validation_stats.py]: Commented out code -- blick!

**Pull request merged**